app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+    @staticmethod
+    def print_alive() -> str:
+        return f'[{", ".join([repr(animal) for animal in Animal.alive])}]'
```

Comment:
No need to create a list inside join, it should work with a generator (just remove `[]`)

Reply:
Done